src/views/anmeldungen/details.vue

```diff
@@ -1,119 +1,151 @@
 <template>
-  <div>
-    <v-toolbar tabs ripple>
-      <v-btn 
-        v-accent-bg dark 
-        @click="$router.back()">
-        <v-icon>keyboard_arrow_left</v-icon>
-        Zurück
-      </v-btn>
-      <v-spacer/>
-      <span v-primary style="font-size: 2.2em !important;" class="headline">
-        Timeout (Sebastian Krüger)
-      </span>
-      <v-spacer/>
-      <ec-edit @click="no"/>
-      <v-tabs v-model="tabs" fixed-tabs color="transparent" slot="extension">
+  <ec-wrapper title="Anmeldung Details" :label="`${data.anmeldung.person.vorname} ${data.anmeldung.person.nachname} (${data.anmeldung.person.gebDat.german}) - ${data.anmeldung.veranstaltung.bezeichnung} (${data.anmeldung.veranstaltung.begin.year}) - ${['Teilnehmer','Mitarbeiter','Küche','Leiter','Hauptleiter'][data.anmeldung.position]}`" type="Anmeldung" @share="share">
+
+    <template slot="label">
+      <ec-headline>
+        {{data.anmeldung.person.vorname}} {{data.anmeldung.person.nachname}} ({{data.anmeldung.person.gebDat.german}}) - {{data.anmeldung.veranstaltung.bezeichnung}} ({{data.anmeldung.veranstaltung.begin.year}}) - {{['Teilnehmer','Mitarbeiter','Küche','Leiter','Hauptleiter'][data.anmeldung.position]}}
```

Comment:
Hier würde reichen VERANSTALTUNG - VORNAME NACHNAME

Alle anderen Informationen in den Tab soriert.

Reply:
Done (aber mit "Rolle") da das für mich da reingehört...